packages/handlersjs-core/lib/daemons/scheduler.ts

```diff
@@ -0,0 +1,59 @@
+import { Observable, Subject } from 'rxjs';
+import { Daemon } from '../models/daemon';
+
+export class Scheduler extends Daemon {
+
+  private currentTimeout: NodeJS.Timeout | undefined;
+
+  constructor(
+    protected readonly interval: number,
+    protected readonly task: (() => void)
```

Comment:
Was there a specific reason for making these `protected readonly`? (I actually think that's a good idea, even though we typically use `private`.)